packages/core/src/pages/index.tsx

```diff
@@ -30,6 +31,7 @@ const COMPONENTS: Record<string, ComponentType<any>> = {
   ProductShelf,
   ProductTiles,
   BannerText,
+  BannerNewsletter,
```

Comment:
I think we could add `CrossSellingShelf` here too

Reply:
I was wondering about this, as this Section is restricted to PDP and LP [from the hCMS sections.json](https://github.com/vtex/faststore/blob/feat/adds-more-sections-to-pages-SFS-909/packages/core/cms/faststore/sections.json#L948), so people can't add it from hCMS so far. I think we can review it in the future.